NuRadioReco/modules/channelBandPassFilter.py

```diff
@@ -47,38 +47,34 @@ def run(self, evt, station, det, passband=[55 * units.MHz, 1000 * units.MHz],
         filter_type='FIR hamming': same as above
         filter_type='FIR kaiser 10' : Use Kaiser window with beta parameter 10
         filter_type='FIR kaiser' : Use Kaiser window with default beta parameter 6
-        
+
         In principal, window names are just passed on to signal.firwin(), but if parameters
         are required, then these cases must be explicitly implemented in the code below.
-        
-        The for main filter types can be implemented: 
```

Comment:
While at it, please fix the langugage in this sentence ;)

Reply:
Done